service/src/main/java/org/databiosphere/workspacedataservice/config/DataImportProperties.java

```diff
@@ -63,6 +64,20 @@ public void setSucceedOnCompletion(boolean succeedOnCompletion) {
     this.succeedOnCompletion = succeedOnCompletion;
   }
 
+  /**
+   * Permissions syncing on TDR import jobs is only enabled on the control plane.
+   *
+   * @see org.databiosphere.workspacedataservice.dataimport.tdr.TdrManifestQuartzJob
+   * @return the configured value
+   */
+  public boolean isPermissionSync() {
```

Comment:
This reads a little awkwardly. Maybe something like:

```suggestion
  public boolean isTdrPermissionSyncingEnabled() {
```

Reply:
I agree, but how married are we on the convention of the get function being is<PropertyName>? That's how this function is named

Reply:
In my opinion, we're not very attached to that convention. TenancyProperties contains some boolean properties with getters named `getProperty`.

And I also proposed renaming the property in the profile YAML 😁.

Reply:
aight, renamed